libavutil/hwcontext_qsv.c

```diff
@@ -1317,8 +1317,6 @@ static int qsv_get_buffer(AVHWFramesContext *ctx, AVFrame *frame)
 
     frame->data[3] = frame->buf[0]->data;
     frame->format  = AV_PIX_FMT_QSV;
-    frame->width   = ctx->width;
-    frame->height  = ctx->height;
 
```

Comment:
qsv_get_buffer is used in qsv based filters too, does this change impact these filters ?